### PR TITLE
Lambda.invoke from another lambda

### PR DESCRIPTION
**Pull request by nelsonic**
Adds basic instructions for how to `invoke` a Lambda function from another Lambda. #68

### CODE REVIEW

**Comment by nelsonic** [comment]
@nikhilaravi / @jackcarlisle / @iteles do either of you have time to review give feedback on this?

File: examples/lambda_invoke/README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  function invokeFunction() {
+    lambda.invoke(params, function(err, data) {
+      if (err) {
+        context.fail(err);
+      } else {
+        context.succeed('Lambda_B said '+ data.Payload);
+      }
+    })
+  }
+  // first check that the
```
Comment (nikhilaravi) [inline examples/lambda_invoke/README.md]: did you mean to finish this sentence?

**Comment by nikhilaravi** [comment]
@nelsonic looks great! Nice simple example :+1:

File: examples/lambda_invoke/README.md (hunk 1 above)
Comment (nelsonic) [inline examples/lambda_invoke/README.md]: thanks for reviewing and spotting the incomplete sentence. I've updated it. :+1:

**Comment by nelsonic** [comment]
@nikhilaravi thanks for reviewing it. I updated the incomplete line so it should be mergeable. :+1:

**Comment by nikhilaravi** [comment]
@nelsonic cool I'll merge it now! :+1:

**Pull request merged**